doc/plugins/org.polarsys.capella.ui.doc/html/First steps with Capella/3.8. How to import Capella Project Example.mediawiki

```diff
@@ -1,15 +1,23 @@
-
-= How to import a Capella Project Example? =
-
-# '''File -> New -> Example ...'''
-# Select '''Capella Project Example'''
-# The wizard dialog appears and proposes a list of available examples. An example decorated by a star means that it was already imported into your workspace, you have to click on the '''Rename''' button in order to be able to import it again. 
-
-[[Image:Images/3.8.%20How%20to%20import%20Capella%20Project%20Example_1.png]]
-[[Image:Images/3.8.%20How%20to%20import%20Capella%20Project%20Example_2.png]]
-
-= How to add an existing Capella project as an example? =
-# The sample project must be a zip containing the project folder
-# The sample project shall be placed in sample folder like '''CAPELLA_INSTALLATION_PATH/samples/model.zip''' or located under a sub folder like '''CAPELLA_INSTALLATION_PATH/samples/model/model.zip'''
-# The import sample wizard dialog will propose a list of available examples including the newly added ones.
-[[Image:Images/3.8.%20How%20to%20add%20new%20Capella%20Project%20Example.png]]
+
+= How to import a Capella Project Example? =
+Sample projects can be imported using File / Import, then Capella Project Example:
+<br><br>
+[[Image:images/ImportCapellaProjectExample1.png|500x500px]]<br><br>

```

Comment:
When we open the mediawiki file on github, the image is not loaded.
https://github.com/eclipse/capella/blob/32c819fbe6b4acc26bc60e6c4adc447fc880e368/doc/plugins/org.polarsys.capella.ui.doc/html/First%20steps%20with%20Capella/3.8.%20How%20to%20import%20Capella%20Project%20Example.mediawiki

I think its probably due to lowercase of `images/` folder instead of `Images/`